<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pickParticipants</class>
 <widget class="QDialog" name="pickParticipants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pickParticipants</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pick Participants</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="driversTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>700</width>
     <height>350</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Pilot</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Team</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>VIN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Reg. no.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Engine Cap.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Engine Pow.</string>
    </property>
   </column>
   <column/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
